Index: Home Page.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Home Page.html b/Home Page.html
--- a/Home Page.html	
+++ b/Home Page.html	
@@ -1,0 +1,23 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<header>
+    class="header">
+  <a href="#" class="logo">Developer</a>
+  <nav class="nav-items">
+    <a href="#">Home</a>
+    <a href="#">Shop</a>
+    <a href="#">About</a>
+      <a href="#">Contact Us</a>
+  </nav>
+</header>
+
+<main>
+  <div class="intro">
+    <h1>AROUND THE BLOCK.CO</h1>
+    <p>Join the Block Party</p>
+    <button>Start Shopping</button>
+  </div>
+<main>
+
+</html>
\ No newline at end of file
